cypress/integration/SearchPage/index.js

```diff
@@ -193,7 +199,7 @@ Then('there should be a validation error message', () => {
 });
 
 Given('you are on the search page with preselected program and org unit', () => {
-    cy.visit('/#/search/programId=VBqh0ynB2wv&orgUnitId=DiszpKrYNg8');
+    cy.visit('/#/search/programId=qDkgAbB5Jlk&orgUnitId=DiszpKrYNg8');
```

Comment:
I change this because before this was a Malaria case Event program. Now this doesnt make much sense